### PR TITLE
Decompme

### PR DESCRIPTION
**Pull request by em-eight**
Add decomp.me scratch creation script

### CODE REVIEW

File: mkwutil/tools/decompme.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    preset = "Mario Kart Wii (DOL)"
+    disaser = get_dol_disaser()
+else:
+    preset = "Mario Kart Wii (REL)"
+    disaser = get_rel_disaser()
+
+# Disassemble function
+fn_start_vma = addr
+fn_end_vma = fn_start_vma + sym.size
+asm, _ = disaser.function_to_text_with_referenced(fn_start_vma, inline=False, extra=True,
```
Comment (SeekyCt) [inline mkwutil/tools/decompme.py:52]: Why not just call function_to_text?

Comment (em-eight) [reply mkwutil/tools/decompme.py:52]: No particular reason. Pushing function_to_text version soon

**Comment by SeekyCt** [comment]
Anything in particular you're wanting me to review? The swapped function call looks fine afaik

**Comment by em-eight** [comment]
Tested this locally too. All good then

**Pull request merged**